docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -64,31 +74,114 @@ Over the duration of a given microsimulation run, simulant attributes are dynami
 Simulation components are interdependent, and relationships between attributes are also based on best available real-world 
 evidence. For example, disease progression is a function of simulant characteristics and treatment; survival depends on stage of disease and other characteristics. 
 
+As the simulation proceeds, various simulant attributes change and these changes are recorded through observers (to read more about observers, visit [insert link here]).
```

Comment:
Do we have a link for this?

Reply:
I don't think so, which makes me wonder if there should be somewhere in the docs that explain what we mean when we say 'observer'? It's pretty intuitive but could be helpful to have something brief somewhere!

Reply:
We have an open documentation ticket to add an observers page to the "types of vivarium model components." Maybe we can write a to-do note here to link to it when it is created?

Reply:
Oh sweet! I'll add a to-do in that case